sdk/attestation/attestation/karma.conf.js

```diff
@@ -122,6 +122,10 @@ module.exports = function(config) {
       ChromeHeadlessNoSandbox: {
         base: "ChromeHeadless",
         flags: ["--no-sandbox", "--disable-web-security"]
+      },
+      ChromeInteractiveNoSandbox: {
+        base: "Chrome",
+        flags: ["--no-sandbox", "--disable-web-security"]
```

Comment:
NIT: Perhaps you no longer need this

Reply:
Actually, I intentionally added it to make debugging the browser easier - this way it's a simpler switch to use the browser.